plone/scale/storage.py

```diff
@@ -14,6 +14,15 @@
 # See AnnotationStorage._keep_scale_seconds instead.
 KEEP_SCALE_MILLIS = 24 * 60 * 60 * 1000
 
+try:
+    long
+except NameError:
+    # Python 3 has no long, only int.
```

Comment:
I have switched to using `six`. With my initial code, a Python 3 version of `flake8` would still complain: `undefined name 'long'`, even though that is caught in the `try/except`.
So `six` for the win!
`flake8` does not complain then, though that does not necessarily mean that this package is Python 3 compatible. Should be fine though.